src/cmdc_tools/datasets/official/AR/data.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import requests
+import textwrap
+
+from ..base import ArcGIS
+
+
+class Arkansas(ArcGIS):
+    ARCGIS_ID = "PwY9ZuZRDiI5nXUB"
+
+    def __init__(self, params=None):
+
+        if params is None:
+            params = {
+                "f": "json",
+                "where": "1=1",
+                "outFields": "*",
+                "returnGeometry": "false",
+            }
+
+        super(Arkansas, self).__init__(params)
+
+    def _insert_query(self, df, table_name, temp_name, pk):
```

Comment:
@cc7768 I'm guessing that this is using the most recent code for `Arkansas` from the master branch? That branch was merged into Master, correct? If this is rebased with the most recent Master, this is good to go.

Reply:
Yep. Everything is on most recent master.